assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -0,0 +1,51 @@
+import classnames from 'classnames';
+const { __ } = wp.i18n;
+
+export const CampaignCovers = ({ covers, initialRowsLimit, row, loadMoreCovers }) => {
+  const amountPerRow = 3;
+  const showLoadMore = !!initialRowsLimit && covers.length > amountPerRow * initialRowsLimit;
+
+  return (
+    <div className='container'>
+      <div className='thumbnail-largeview-container'>
+        {covers.map((cover, index) => {
+          const { href, image, alt_text, name, src_set, sizes } = cover;
+          const hideCover = !!initialRowsLimit && index >= row * amountPerRow;
+          return (
+            <div key={name} className={classnames('campaign-card-column', { hidden : hideCover })}>
+              <a
+                href={href}
+                data-ga-category='Campaign Covers'
+                data-ga-action='Image'
+                data-ga-label='n/a'
+                aria-label={__('Check our campaign about ' + name, 'planet4-blocks')}
+              >
+                <div className='thumbnail-large'>
+                  {image && image[0] &&
+                    <img
+                      loading='lazy'
+                      sizes={sizes}
+                      srcSet={src_set}
+                      src={image[0]}
+                      alt={alt_text}
+                    />
+                  }
+                  <span className='yellow-cta'><span aria-label='hashtag'>#</span>{name}</span>
+                </div>
+              </a>
+            </div>
+          );
+        })}
+      </div>
+      {showLoadMore && (row * amountPerRow) < covers.length &&
```

Comment:
Doesn't this kinda check the same thing twice? :thinking:

Reply:
Right, since `row` is at first set to `initialRowsLimit` this is unnecessary 🙂 Fixed!